src/bundles/ApplicationsAdmin/components/EvidenceAssessment/EvidenceAssessment.js

```diff
@@ -165,9 +165,10 @@ class EvidenceAssessment extends React.Component {
 
   render() {
     const { evidence } = this.props
-    const essentialCriteriaIds = evidence.domain_criteria.filter(
-      criterion => criterion.essential
-    ).map(criterion => criterion.id)
+    let essentialCriteriaIds;
```

Comment:
Should we initialise to an array since code below expects an array?
```suggestion
    let essentialCriteriaIds = [];
```